dd-trace-ot/src/main/java/datadog/trace/common/writer/ZipkinV2Api.java

```diff
@@ -91,14 +92,14 @@ public Response sendSerializedTraces(
         int traceCount = 0;
 
         out.write('[');
```

Comment:
This has always made me feel a little dirty.  Would using the streaming api be better in your opinion?

Reply:
I wouldn't lose any sleep over it; this code is EOL eventually anyway.